modules/deployment/src/test/java/org/mule/runtime/module/deployment/internal/AbstractDeploymentTestCase.java

```diff
@@ -254,6 +256,13 @@ public static void beforeClass() throws URISyntaxException {
         new SingleClassCompiler().compile(getResourceFile("/org/foo/LoadsAppResourceCallback.java"));
     pluginEcho1TestClassFile =
         new SingleClassCompiler().dependingOn(barUtils1_0JarFile).compile(getResourceFile("/org/foo/Plugin1Echo.java"));
+
+    FieldUtils.writeDeclaredStaticField(ModuleDelegatingEntityResolver.class, "internalIsRunningTests", true, true);
+  }
+
+  @BeforeClass
+  public static void afterClass() throws IllegalAccessException {
+    FieldUtils.writeDeclaredStaticField(ModuleDelegatingEntityResolver.class, "internalIsRunningTests", false, true);
```

Comment:
get the previous value in `before` and reset that here instead of `false`. Maybe use a `@Rule`